### PR TITLE
Implement portable variants of TypedArray.wrap

### PR DESCRIPTION
**Pull request by jletellier**
Implements #1304 

- [x] I've read the contributing guidelines

### CODE REVIEW

File: std/portable/index.js [hunk 1]
```diff
@@ -313,3 +313,57 @@ globalScope["trace"] = function(message, n) {
   if (n) message += Array.prototype.slice.call(arguments, 2, 2 + n);
   console.error("trace: " + message);
 };
+
+Object.defineProperty(Int8Array, "wrap", {
+  value: function(buffer, byteOffset, length) {
```
Comment (MaxGraey) [inline std/portable/index.js:318]: could you plz add function names?
```suggestion
  value: function wrap(buffer, byteOffset, length) {
```
And for rest cases as well

Comment (jletellier) [reply std/portable/index.js:318]: Absolutely; I can't believe I missed that.

**Comment by MaxGraey** [comment]
@dcodeIO wdyt should it also optionally handle `BigInt64Array` and `BigUint64Array`?

**Comment by dcodeIO** [comment]
Hmm, good question. There is no good i64 story yet, and portable doesn't yet try to polyfill any of that, including the basic type. I suggest leaving the PR as-is for now, and think about better portable i64 support in general later, for example we'd also have to alias `BigInt64Array` to `Int64Array` first.

**Comment by dcodeIO** [comment]
Perhaps on a second note, one way to polyfill i64 in general might be to highjack the i64.xy set of instructions, so one can write `i64.add(x, y)` etc. in both JS and Wasm to make it work. That would also fit well into the compiler I guess, with an `i64` being a Long.js object in JS. However, there are no inliner-assembler variants for very basics ops yet, like the mentioned `i64.add`, but that shouldn't be too hard to add (just a lot of work to add all of them for i32, f32, f64 etc.).

**Pull request merged**